obspy/io/reftek/core.py

```diff
@@ -241,10 +241,14 @@ def to_stream(self, network="", location="", component_codes=None,
             else:
                 eh = EHPacket(et_packets[0])
             # only "C0" encoding supported right now
-            if eh.data_format != b"C0":
+            if eh.data_format == b"C0":
+                encoding = 'C0'
+            elif eh.data_format == b"C2":
+                encoding = 'C2'
```

Comment:
This is a bit awkward to be honest. Here its converted to bytes and in the `_unpack_C0_C2_data()` function its converted back to bytes.